indico/web/client/js/react/components/Checkbox.module.scss

```diff
@@ -0,0 +1,55 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@use 'partials/icons';
+@use 'partials/sui_debt';
+
+.checkbox-label {
+  display: inline-flex;
+  gap: var(--content-gap-normal);
+  align-items: center;
+}
+
+.checkbox {
```

Comment:
I had a look in the room booking admin UI where we use quite a few FinalCheckbox components w/ a description below, and the margins seem to be a bit different from other fields. Could you try to align them a bit more with what we have for those other fields? In the screenshot (the page URL is `/rooms/admin` if you want to go there yourself) it's quite easy to see:

![image](https://github.com/user-attachments/assets/6f9d44f4-26da-4cfe-8108-8ec548878532)

Reply:
Will check